### PR TITLE
Config updates

### PR DESCRIPTION
**Pull request by Tiagoquix**
**mm_override.cfg**

- Removed some commands that the config doesn't set anymore and added new ones to prevent crashes.

**comfig.cfg**

- Fixed inconsistency with `r_shadowlod -2` being globally set, because using `restore_config` (or joining spectator before joining any team in your first match) with `shadows=ultra` won't set `r_shadowlod 0` until a new class change.
- Fixed inconsistency with `r_lightcache_zbuffercache` being globally set to `1` but `lighting`'s levels `very_low` and `ultra` setting it to `0`.

### CODE REVIEW

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 1]
```diff
@@ -275,13 +275,13 @@ r_hunkalloclightmaps 0 // Use native memory allocation because z buffer cache ta
 
 alias lightmaps_off"toggle mat_filterlightmaps"
 
-alias lighting_very_low"mat_filterlightmaps 0;alias lightmaps_override lightmaps_off;r_ambientboost 0;r_lightaverage 0;r_dynamic 0;r_maxdlights 0;r_worldlightmin 999999;r_lightcache_zbuffercache 0;r_worldlights 0;mat_disable_lightwarp 1;mat_reducefillrate 1;r_rimlight 0;mat_bumpmap 0;mat_specular 0;building_cubemaps 1;alias lighting_level echo lighting=very_low"
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: It's globally set to 1 for None Preset. The modules intentionally override this, as there is a loss in lighting accuracy with it on, so it is disabled on ultra, and for very low, there is no lighting except from the sun which cannot be cached.

File: config/mastercomfig/cfg/comfig/mm_override.cfg [hunk 2]
```diff
@@ -48,13 +49,12 @@ r_flashlightscissor 1
 r_flex 1
 r_lightcache_zbuffercache 0
 r_lod -1
+r_staticprop_lod -1
 r_maxdlights 32
 r_maxmodeldecal 50
-r_pixelfog 1
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/mm_override.cfg:53]: Don't remove these.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 3]
```diff
@@ -311,19 +311,19 @@ r_flashlightculldepth 0 // Skip more flashlight processes which are ultimately u
 r_flashlightscissor 0 // ^
 //r_shadowmaxrendered 12 // How many high quality shadows are rendered
 //nb_shadow_dist 400 // Distance for NextBot entity shadows
-r_shadowlod -2 // Use the lowest quality model to render shadows
+//r_shadowlod -2 // Use the lowest quality model to render shadows
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: This is for the None Preset.

File: config/mastercomfig/cfg/comfig/comfig.cfg [hunk 4]
```diff
@@ -311,19 +311,19 @@ r_flashlightculldepth 0 // Skip more flashlight processes which are ultimately u
...
-alias shadows_low"r_shadows 1;r_shadowrendertotexture 0;cl_blobbyshadows 0;r_shadowmaxrendered 3;nb_shadow_dist 0;alias shadowlod_override;alias shadows_level echo shadows=low"
-alias shadows_medium"r_shadows 1;r_shadowrendertotexture 1;cl_blobbyshadows 0;r_shadowmaxrendered 7;nb_shadow_dist 100;alias shadowlod_override shadowlod_low;alias shadows_level echo shadows=medium"
-alias shadows_high"r_shadows 1;r_shadowrendertotexture 1;cl_blobbyshadows 0;r_shadowmaxrendered 11;nb_shadow_dist 400;alias shadowlod_override shadowlod_low;alias shadows_level echo shadows=high"
-alias shadows_very_high"r_shadows 1;r_shadowrendertotexture 1;cl_blobbyshadows 0;r_shadowmaxrendered 23;nb_shadow_dist 1000;alias shadowlod_override shadowlod_low;alias shadows_level echo shadows=very_high"
-alias shadows_ultra"r_shadows 1;r_shadowrendertotexture 1;cl_blobbyshadows 0;r_shadowmaxrendered 23;nb_shadow_dist 1000;alias shadowlod_override shadowlod_high;alias shadows_level echo shadows=ultra"
+alias shadows_off"r_shadows 0;r_shadowrendertotexture 0;cl_blobbyshadows 1;r_shadowmaxrendered 0;nb_shadow_dist 0;shadowlod_low;alias shadowlod_override shadowlow_low;alias shadows_level echo shadows=off"
```
Comment (mastercoms) [inline config/mastercomfig/cfg/comfig/comfig.cfg]: There is no need to set shadowlod here. It's only used for render to texture shadows.

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by Tiagoquix** [comment]
Done @mastercoms 

Alternatively, you could move the lightcache and shadowlod commands to the none.cfg preset file, so they don't get executed twice, but that would be unnecessary.

**Comment by mastercoms** [comment]
It's also for people aliasing out modules.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**